grpc-proto/proto/query.proto

```diff
@@ -99,36 +101,39 @@ message Value {
     // CQL types: ascii, varchar, text
     string string = 7;
 
-    // CQL types: blob, inet, custom
+    // CQL types: blob, custom
     bytes bytes = 8;
 
+    // CQL types: inet
+    Inet inet = 9;
```

Comment:
why do we add it in the middle (instead of at the end)?

Reply:
We should favor types that are more likely to be used in the first "1 thorugh 15" field numbers: https://developers.google.com/protocol-buffers/docs/proto3#assigning_field_numbers

Also, it's okay to break the contract currently.